### PR TITLE
Send mail

### PR DESCRIPTION
**Pull request by orovinskiy**
Finished the rough bones of the send page. Just waiting on chad to finish his part closes issue #51

### CODE REVIEW

File: php/mailSender.php [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            echo "<p>From: $subject</p>";
+        }
+
+        if(!validReqText($compose)){
+            echo "<p>Error: The Compose field was left empty</p>";
+            $isValid = false;
+        }
+        else{
+            echo "<p>From: $compose</p>";
+        }
```
Comment (grjk) [inline php/mailSender.php]: All three else statements echo "From:", but should be "From:", "Subject:" and "Compose/Message/Etc:".

File: php/mailSender.php [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        if($isValid){
+            $emailSend = $from;
+
+            $email_body = "I Day Dream --\r\n";
+            $email_body .= "$compose \r\n";
+
+
+
+            $email_subject = $subject;
+            $to = "olegrovin@gmail.com";//The active user goes here
```
Comment (grjk) [inline php/mailSender.php]: I tried this with my email but never got the email. Everything is uploaded on cpanel, not sure if it should be sending or not.

**Comment by grjk** [comment]
Also, it doesn't validate the email field for "From" correctly. You could type in anything you want without putting an "@address.com" and it will work.

File: php/dreamers.php [hunk 3]
```diff
@@ -107,6 +107,8 @@
         ?>
         </tbody>
     </table>
+    <br>
+    <p><a href="../sendMail.php?source=dream">Send Email</a></p>
```
[the remark this answers is not in the document]
Comment (orovinskiy) [reply php/dreamers.php]: for volunteer pending will be zero and inactive will be -1

File: sendMail.php [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    <div class="jumbotron banner">
+        <h1>Compose Your Email</h1>
+    </div>
+    <form id="emailComposer" action="php/mailSender.php" method="post">
+        <!-- From -->
+        <div class="form-group">
+            <label class=" h5 font-weight-light " id="fromLabel"  for="from">From</label>
+            <span id="fromError" class="hidden error">*Required</span>
+            <input class="required form-control" type="text" id="from" maxlength="50" name="from">
+        </div>
```
[the remark this answers is not in the document]
Comment (orovinskiy) [reply sendMail.php]: I'll leave it in for now and we will ask her tomorrow

**Pull request merged**